docs/sphinx/source/changelog/pending.rst

```diff
@@ -0,0 +1,23 @@
+************************
+Pending
+************************
+
+Testing
+-------
+* Added a CI notebook check (:pull:`270`)
+
+Requirements
+------------
+* Upgrade the notebook environment from python 3.7 to python 3.10.
+  Several dependency versions in ``docs/notebook_requirements.txt`` are
+  updated as well. (:issue:`319`, :pull:`326`)
+* Bump ``ipython==7.16.3``, ``jupyter-console==6.4.0``,
+  and ``prompt-toolkit==3.0.27`` in ``docs/notebook_requirements.txt``
+  and bump ``Pillow==9.0.0`` in ``requirements.txt`` (:pull:`314`)
+* Bump ``sphinx`` version from 3.2 to 4.5 and ``nbsphinx`` version
+  from 0.8.5 to 0.8.8 in the optional ``[doc]`` requirements (:pull:`317`, :pull:`325`)
+* A number of other requirements updates (:pull:`337`)
```

Comment:
I think these old `pending` entries are already covered in `v2.2.0-beta.0.rst`; let's clean out everything but the entry for this PR

Reply:
Updated to just this PR in pending.rst in commit https://github.com/NREL/rdtools/pull/348/commits/c3838d65fc6a0b71a961455ff6a36641c7054cc7